src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -348,7 +348,7 @@
     <Compile Include="FlowAnalysis\AlwaysAssignedWalker.cs" />
     <Compile Include="FlowAnalysis\ControlFlowAnalysis.cs" />
     <Compile Include="FlowAnalysis\ControlFlowPass.cs" />
-    <Compile Include="FlowAnalysis\DataFlowAnalysis.cs" />
+    <Compile Include="FlowAnalysis\CSharpDataFlowAnalysis.cs" />
```

Comment:
Is there a particular reason for the rename? It feels strange to use CSharp prefix for files in CSharp folder.